service/src/main/java/com/ge/predix/acs/audit/PredixEventMapper.java

```diff
@@ -0,0 +1,44 @@
+package com.ge.predix.acs.audit;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+
+import com.ge.predix.audit.sdk.message.AuditEnums.CategoryType;
+import com.ge.predix.audit.sdk.message.AuditEnums.Classifier;
+import com.ge.predix.audit.sdk.message.AuditEnums.EventType;
+import com.ge.predix.audit.sdk.message.AuditEnums.PublisherType;
+import com.ge.predix.audit.sdk.message.AuditEventV2;
+import com.ge.predix.audit.sdk.message.AuditEventV2.AuditEventV2Builder;
+
+public class PredixEventMapper {
+
+    public AuditEventV2 map(final com.ge.predix.audit.AuditEvent auditEvent) {
+        Map<String, String> auditPayload = new HashMap<String, String>();
+        auditPayload.put("httpMethod", auditEvent.getMethod());
+        auditPayload.put("requestBody", auditEvent.getRequestBody());
+        auditPayload.put("responseBody", auditEvent.getResponseBody());
+
```

Comment:
Should we create a json string out of auditPayload and then store it in payload field.